### PR TITLE
Mobx5 Support

### PR DESCRIPTION
**Pull request by chuckcfs**
- Updated project dependencies to use `mobx` and `mobx-react` latest
version ^5
- Modified `LocaleStore` constructor to wrap the `_locale` creation
inside an observable box call to comply with Mobx5 primitives
observables creation

### CODE REVIEW

**Comment by ChrisJamesC** [comment]
Thanks for the PR. Could you please fix the CircleCI issues?

File: package.json [hunk 1]
```diff
@@ -119,8 +119,8 @@
     "format-message": "^5.2.0"
   },
   "peerDependencies": {
-    "mobx": "^3.0.0",
```
Comment (ChrisJamesC) [inline package.json:122]: Do we have to not make the code compatible with older versions of mobx?

**Comment by ChrisJamesC** [comment]
Fixes #19

**Comment by chuckcfs** [comment]
hey, sorry for the waaaaay too late reply, I'm just seeing this and checked the circleci issues but they're related to mobx, something wrong in their code I think

**Pull request closed without merging**